### ISSUE TITLE
MW72 IAST:  use modern forms only

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/MW72#4]
As mentioned in #3,  the recent conversion to IAST for MW72 intentionally left certain non-modern IAST forms in place.  This was done to facilitate comparison work via the [UI](http://www.sanskrit-lexicon.uni-koeln.de/scans/MW72Scan/2014/pywork/correctionwork/issue-320/307/update.php).

However, I think that it would be better to go all the way to modern IAST for MW72, as we are doing with
other dictionaries.

The items marked with `*` in [this table](https://github.com/sanskrit-lexicon/MW72/blob/master/20161107/mwiast-iast.txt)  identify places where changes would be made.

For instance `f ṛi ṛ *`  in the table means that the SLP1 letter 'f' is represented in MW72 currently as `ṛi` which would
change to `ṛ ` in modern IAST.

The change I'm proposing would replace, in mw72.txt,  the 2nd form with the third in all the asterisked cases.

**Comment by gasyoun** [comment]
> For instance f ṛi ṛ * in the table means that the SLP1 letter 'f' is represented in MW72 currently as ṛi which would
change to ṛ in modern IAST.

Hurray!

**Comment by drdhaval2785** [comment]
After IAST conversion of all dictionaries, I see no trace of `ṛi` in mw72.txt except in the preface where it occurs in the word 'Sanskṛit' universally. @funderburkjim, is it intentional or accidental?

**Comment by drdhaval2785** [comment]
Example from the frontmatter is as follows
```
<P>1. Reasons for undertaking a New Sanskṛit Dictionary.
<P>2. Plan and Arrangement of the Present Work.
<P>3. Extent of Sanskṛit Literature comprehended.
<P>4. Alphabet and System of Transliteration employed.
<P>5. Principal Sources drawn upon in the Process of Compilation.
<P>6. Aids and Encouragements received.
<P>7. Defects and Inconsistencies acknowledged.
;
; PREFACE: Reasons for undertaking a New Sanskṛit Dictionary.
```

**Comment by funderburkjim** [comment]
All of these 'ṛi' instances are in the front matter (i.e., before the front entry at line 1735
`<L>1<pc>0001-a<k1>a<k2>a<h>1`.

The programmatic changes I made did not apply to this front matter part of mw72.txt.

Within the entries of mw72, the spelling of `Sanskṛit` has been changed to `Sanskrit`.

We could change the spelling in the preface part of mw72.txt,  but currently this part of
mw72 is not used anywhere.

Up to you.

But if you do change mw72.txt,  beware the possible problem with sqlite3 currently in place
at Cologne.
